sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -567,7 +567,7 @@ abstract class DataWritingCommandMeta[INPUT <: DataWritingCommand](
     if (types.exists(GpuOverrides.isOrContainsTimestamp(_))) {
       if (!GpuOverrides.isUTCTimezone()) {
         willNotWorkOnGpu("Only UTC timezone is supported for " +
-          "GpuCreateHiveTableAsSelectCommand. " +
+          s"${wrapped.getClass}. " +
```

Comment:
The class gets printed out in different ways in different places. I think it would probably be better to just remove this part.

"Only UTC timezone is supported. " +
...